packages/projects/ProjectConfigInterface.ts

```diff
@@ -67,4 +69,10 @@ export interface ProjectEventHooks {
   onUninstall?: InstallFunctionType
 }
 
+export interface ProjectSettingSchema {
+  key: string
+  type: string
+  scopes: Array<string>
```

Comment:
not being used?

Reply:
this will be used when we start adding code to check whether a person has access to any project setting or not